scripts/windows/wix-populate_internal_artifacts.py

```diff
@@ -0,0 +1,59 @@
+"""Add all components (files) and directories generated by pyinstaller in
+dist/papis/_internal into the XML manifest that is used to create the MSI."""
+
+import os
+import lxml.etree as et
+import uuid
+
+
+def get_id(*path_components: str) -> str:
+    """Create an element ID based on the WiX constraints:"""
```

Comment:
Is there a place describing the "WiX constraints"? Would be nice to link it here for future reference.

Reply:
definitely!